src/lib/config.js

```diff
@@ -202,6 +203,15 @@ function getLocalConfig () {
   // }
   const ledgerCredentials = parseCredentials()
 
+  const secretString = Config.getEnv(envPrefix, 'SECRET')
```

Comment:
you can do all this as `const secret = Buffer.from(Config.getEnv(envPrefix, 'SECRET'), 'base64') || crypto.randomBytes(32)`